operator/src/test/java/oracle/kubernetes/operator/DomainWatcherTest.java

```diff
@@ -43,6 +45,13 @@ public void initialRequest_specifiesStartingResourceVersion() {
         hasEntry("resourceVersion", INITIAL_RESOURCE_VERSION.toString()));
   }
 
+  @Test
```

Comment:
I don't understand the name of this test. It seems to be testing that if a watcher receives  a bookmark callback, it updates its resource version for future queries. Is that correct? If so, I'd suggest that the test be named something like, `whenWatcherReceivesBookmarkEvent_updateResourceVersion`. The current name seems to be talking about what Kubernetes does, not what the operator does.

Reply:
Changed